styles/porkbun/catppuccin.user.css

```diff
@@ -248,6 +248,10 @@
       background-color: @mantle;
       color: @text;
       border-color: @surface0;
+
+      &:focus {
+         box-shadow: 0 0 8px fade(@accent-color, 50%);
```

Comment:
Not the box shadow I'm seeing...

Reply:
is it unthemed or is the focus different on your side?

Reply:
Doesn't match the box shadow line they use.

Reply:
> Doesn't match the box shadow line they use.

isn't it ``box-shadow: 0 0 8px rgba(203, 166, 247, 0.5);``?

Reply:
Ah I stand corrected. Must have been looking at the wrong thing yesterday.

Reply:
also i followed your feedback and used less's functions along with ``fade``